product_docs/docs/ocl_connector/15/ocl_rel_notes/15.2.0.3_ocl_release_notes.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Version 15.2.0.3"
+---
+The EDB OCL Connector provides an API similar to the Oracle Call Interface.
+
+New features, enhancements, bug fixes, and other changes in the EDB OCL Connector 15.2.0.3 include:
+
+|  Type   |                                                                       Description                                                                        |
+| ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Bug fix | Fixed regression whereby tnsnamesparse() tries to parse tnsnames.ora, but reads it from stdin (fd 0) instead of the file. [Support tickets: 93920, 93994] |
```

Comment:
small thing, but maybe code formatting for fn name?
```suggestion
| Bug fix | Fixed regression whereby `tnsnamesparse()` tries to parse tnsnames.ora, but reads it from stdin (fd 0) instead of the file. [Support tickets: 93920, 93994] |
```